docs/book/localization.md

```diff
@@ -464,52 +462,213 @@ backslash:
+
+In the process of skill development, you might revise your default skill code
+strings at the same time you are adding localizations to your skill. To help
+you keep track of changes, the `localize` command will output a summary of the
+`localization.json` strings that have changed.
+
+It's important to remember that the `localize` command reads strings from
+*only* the default skill code. To keep things straight, you can keep in mind
+these two truths as you localize:
+
+* the default skill code is the source of truth for what utterances and
+  say/reprompt strings to localize
+* `localization.json` is the source of truth for what's covered for localized
+  languages
+
+Here's an example on what would happen if you had modified a string after it
+was localized. Your Litexa project initially looks like this, after running
+`litexa localize` and adding translations for Spanish:
+
+```coffeescript
+# main.litexa
+launch
+  say "Hello."
+    or "Hi."
+  say "What's your name?"
+```
+
+```json
+// localization.json
+"speech": {
+  "Hello.|Hi.": {
+    "es": "Hola."
+  },
+  "What's your name?": {
+    "es": "¿Cómo se llama?"
+  }
+}
+```
+
+If we then changed `main.litexa` to this:
+
+```coffeescript
+# main.litexa
+launch
+  say "Howdy."
+  say "What's your name?"
+```
+
+Running `litexa localize` would then output this in the console:
+
+```stdout
+[localization] +441ms parsing default skill intents, utterances, and output speech ...
+[localization] +320ms the following speech lines are new since the last localization:
+[localization] +1ms + Howdy.
+[localization] +0ms the following speech lines in localization.json are missing in skill:
+[localization] +0ms - Hello.|Hi.
+```
+
+And `localization.json` would now have:
+
+```json
+"speech": {
+  "Howdy.": {}, // new line
+  "What's your name?": {
+    "es": "¿Cómo se llama?"
+  },
+  "Hello.|Hi.": { // orphaned line
+    "es": "Hola."
+  }
+}
+```
+
+So, if the translations from the original string still apply, they should be
+copied to the new string mapping manually.
+
+Intents and utterances changed in the same way follow the same behavior.
+
+### `litexa localize` arguments
 
 `litexa localize` can receive some arguments to modify its behavior. To see the
 complete list, run `litexa localize --help`.
 
-#### Cloning
+#### Verbose logs
+
+The `--verbose` flags turns on logging to give more detail on the contents that
+have changed from the existing `localization.json`.
+
+#### Auto-removing orphaned strings
+
+Orphaned speech strings and orphaned utterances can be automatically removed
+with the `--remove-orphaned-speech` and `--remove-orphaned-utterances` flags,
+respectively, but note that doing so will also remove their translations from
+`localization.json`.
+
+#### Cloning translations
 
-With cloning, the arguments for cloning your default language to French would look like:
+With cloning, the arguments for cloning your default language to French would
+look like:
```

Comment:
Outdated. Would suggest merging this with the paragraph after the code sample to something like:

The existing translations for one language (e.g. fr-FR) can be cloned to another language (e.g. fr-CA) which is useful as a starting point for regionalizing a translation with the same base language: